apps/web/middleware.ts

```diff
@@ -196,8 +204,6 @@ export const config = {
     "/routing-forms/:path*",
     "/team/:path*",
     "/org/:path*",
-    "/:user/:type/",
-    "/:user/",
```

Comment:
Imagine, `/icons/sprite.svg` comes through middleware because it matches `"/:user/:type/"`, and we rewrite this to `/en/icons/sprite.svg`, leading to 404.

TODO: Make sure this doesn't bring back the Vercel timeouts